### PR TITLE
Feature/add marks

### PR DESCRIPTION
**Pull request by ch1ll0ut1**
add an `addMarks(marks)` change #1283

### CODE REVIEW

**Comment by ianstormtaylor** [comment]
Hey @ch1ll0ut1 thanks for this, it looks great!

Do you think the `addMarks()` change could also just loop `marks` and call into `addMark()` instead? Or did you try that already and there was an issue? Thanks!

**Comment by ch1ll0ut1** [comment]
Hi @ianstormtaylor,
there was no real issue but my though was to create a performant loop but after looking at addMarks i wasn't sure that would be so easy. For the moment i let it in so that someone could easy replace that one line with a more performant version and does not have to change any other code as long as it is using addMarksAtRange. Also i think it makes the api more consistent by providing the same kind of method throughout the api. 

But anyway I also wanted to know what you think. I don't mind if you want me to take it out.

File: .yarnrc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+workspaces-experimental true
```
Comment (ianstormtaylor) [inline .yarnrc]: Could you remove this file? I think with `yarn@1.x` this is no longer required.

File: packages/slate/src/changes/at-current-range.js [hunk 2]
```diff
@@ -73,6 +73,40 @@ Changes.addMark = (change, mark) => {
 }
 
 /**
+ * Add a list of `marks` to the characters in the current selection.
+ *
+ * @param {Change} change
+ * @param {Mark} mark
+ */
+
+Changes.addMarks = (change, marks) => {
```
Comment (ianstormtaylor) [inline packages/slate/src/changes/at-current-range.js]: Could this method's logic be implemented like the `addMarksAtRange` one you've added where it simply loops `marks.forEach( ... )` and calls into the old `addMark`? That way we don't end up with two almost-identical implementations to keep in sync?

File: .yarnrc (hunk 1 above)
Comment (ch1ll0ut1) [reply .yarnrc]: removed

File: packages/slate/src/changes/at-current-range.js (hunk 2 above)
Comment (ch1ll0ut1) [reply packages/slate/src/changes/at-current-range.js]: done.

Comment (ch1ll0ut1) [reply packages/slate/src/changes/at-current-range.js]: sry for misunderstanding. Didn't read proper.

Comment (ch1ll0ut1) [reply packages/slate/src/changes/at-current-range.js]: now locally i have no failing tests but circleci has the new tests all failing with:  
Error: No hyperscript creator found for tag: "state"
Do you have a idea what is the problem?

Comment (ianstormtaylor) [reply packages/slate/src/changes/at-current-range.js]: @ch1ll0ut1 Thank you! If you rebase against master you should probably see the error. In the newest versions of Slate, the `state` has been renamed to `value`.

Comment (ch1ll0ut1) [reply packages/slate/src/changes/at-current-range.js]: fixed

**Comment by ianstormtaylor** [comment]
Awesome, thanks @ch1ll0ut1!

**Pull request merged**